server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -1412,7 +1412,7 @@ private void initializeRepoGenerationTracking(ActionListener<RepositoryData> lis
                     }
                     existingListener.onFailure(e);
                 };
-                threadPool.generic().execute(() -> doGetRepositoryData(
+                threadPool.generic().execute(ActionRunnable.wrap(
```

Comment:
Not strictly necessary, just for good measure. We do use `.wrap` in all other calls to `doGetRepositoryData` as well, just missed it here ...